test/fixtures/s3-fixture/src/main/java/fixture/s3/S3HttpHandler.java

```diff
@@ -207,8 +207,9 @@ public void handle(final HttpExchange exchange) throws IOException {
                 exchange.sendResponseHeaders(RestStatus.OK.getStatus(), -1);
 
             } else if (request.isListObjectsRequest()) {
-                if (request.queryParameters().containsKey("list-type")) {
-                    throw new AssertionError("Test must be adapted for GET Bucket (List Objects) Version 2");
+                final var listType = request.queryParameters().getOrDefault("list-type", List.of("1")).get(0);
```

Comment:
Also, injecting "1" is a little misleading: it made me look around for what "1" could represent and it appears to be meaningless?

Might it be better to add a not-null check to the middle of the if-statement?